### DISCUSSION TITLE
Cara menggeser table pada Shiny

### DISCUSSION THREAD

**Discussion by yohanespm77** [teamalgoritma/community#579] [Q&A] [answered]
Saya mau bertanya, bagaimana caranya agar ketika mau menggeser tabel ke kanan/kiri, dashboardnya tidak ikut bergeser. 

<img width="911" alt="image" src="https://user-images.githubusercontent.com/106010284/221476224-2ee6ca54-8372-43a1-8b7e-1d4e032f9fae.png">

Kode saya seperti ini :

#UI
<img width="434" alt="image" src="https://user-images.githubusercontent.com/106010284/221476672-8fa9f1e5-d75d-4ab7-9abe-9359c426a1b6.png">

#SERVER
<img width="460" alt="image" src="https://user-images.githubusercontent.com/106010284/221476705-7e980bf5-389d-48af-863b-0224241d3a86.png">

Mohon bantuannya kak, terima kasih.

**Comment by idoali** [comment, marked as answer]
Terima kasih atas pertanyaannya pak

Bapak bisa membuat dataframe yang bisa digeser kesamping dengan menggunakan fungsi `datatable` dari library `DT`. Pada kasus bapak, pada file `server.R`,  dataframe `input_data` bisa dimasukkan ke dalam fungsi `datatable` kemudia kita atur salah satu `option=` yaitu `scrollX`. 

Kurang lebih akan seperti ini pak : 

```{R}
output$table <- renderDataTable({
        ### Masukkan kode bapak yang lain disini

        datatable(input_data,
                  options = list(scrollX = TRUE))
    })
```

**Reply by yohanespm77** [reply]
Baik, terima kasih Mas Ido.

**Reply by idoali** [reply]
Baik, sama-sama pak
